packages/fxa-settings/src/components/App/RememberPassword/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react';
+import { Localized } from '@fluent/react';
+import { useNavigate } from '@reach/router';
+import { HomePath } from '../../../constants';
+
+type RememberPasswordProps = { email?: string; forceEmail?: string };
+
+const RememberPassword = ({ email, forceEmail }: RememberPasswordProps) => {
+  const navigate = useNavigate();
+
+  const linkTarget: string = email ? '/signin' : '/force_auth';
+  const queryParam: string = email || forceEmail || '';
+
+  const handleClick = useCallback(() => {
+    navigate(HomePath + linkTarget + `?email=` + queryParam);
+  }, [navigate, linkTarget, queryParam]);
+
+  return (
+    <div className="text-sm mt-6" data-testid="remember-password-container">
+      <Localized id="pw-remember-pw">
+        <button
+          onClick={handleClick}
+          id="remember-password"
+          data-testid="remember-password-link"
```

Comment:
Should this be called `button` instead of `link`? (btw  if this was a link I'd recommend calling the component `LinkRememberPassword`)

Reply:
Gave this some thought and I think this does make more sense as a link. I'll change both the element and name.